CharacterCreation/01_Races.txt

```diff
@@ -203,9 +205,47 @@ Once at level 3, and every 5 levels thereafter a Cyborg may purchase an
 have this augment installed.
 
+//============================================================
+Opaleite
+
+Opaleites are so named by the humans who encountered them for the way the light diffracts in their large irises, giving the impression of an opal. A wide-eyed amphibious species that prefers to live entirely in the water. They come in a diverse set of body types and colors depending on where they live and which family they come from. The carry a double set of gills, one set in the neck and one set beneath the lungs that can be closed and sealed, much like the ears and nostrils of aquatic mammals. Unlike their mammalian counterparts, these people can function both underwater and above water. When in the water, the lungs merely act as a bypass for the water running through the gills and gives it a way to exit; while outside of the water, the gill flaps close and the lungs function as mammals’ do. They are incredibly sensitive to sunlight, and burn very easily when not in the water. 
+    Native to the once Ocean planet Trioch II, these beings evolved to the changing of their planet as a massive solar flare from their star shifted the magnetic fields holding in the atmosphere were shaken and ultimately disrupted, causing the oceans to begin evaporating. With their oceans slowly dissipating through the leak in the atmosphere, they have begun migrating to the nearby Trioch IV, also mostly oceanus, which is out of immediate danger from the star. Now, they live in both the oceans and the well-shaded estuaries on Trioch IV. A few stubborn stragglers remain on Trioch II, which is slowly becoming a sandy, dry wasteland with very little atmosphere. 
+
+
+Trioch II is now a place for venturing archeologists to come and investigate the way the Opaleites used to live. 
+
+Racial Bonuses:
+Waterbreathing
++1 base FOR, +10 to reflex rolls. Due to their rapid evolution, Opaleites have a much easier time adjusting to the physical obstacles presented to them. 
+-10 to perception rolls above water.
+Sensitivity to Smoke and Gasesous attacks. -20 to save rolls involving these attacks. 
+
+        Subraces:
+
+Seadevil
+-1 to base CHA, -1 to base FOR.
++1 to DEX
+Low-light vision.
+Ability: Bioluminesce - Once per day.[Is an available skill] Persuade a being, sentient or not, that you are trustworthy. Victim must roll a will-save 50+(Siren points). Effect is dissipated when victims eyes are covered or lose sight of you. This takes a tremendous amount of effort and you take a -10 penalty if tired or bloodied. This ability will temporarily scramble a robot’s visual cortex, but they can adjust rapidly.
+
+Chondrich
+-1 to base CHA, +1 to both STR and FOR
+Tough Skinned: Natural Armor protects you in a firefight. +10 armor coverage, + 5+(FOR/2) Damage reduction. Armor piercing rounds ignore these bonuses. 
+Large: You are larger than your counterparts and are thus easier to hit, -5 DC on any chance to hit you.
+
+Mendar
++2 to base CHA
+-2 to chosen stats.

```

Comment:
-2 to 1 chosen stat? Or a -1 to 2 different stats?

Reply:
It does say stats plural. Is there a balance issue there that I'm not seeing?

Reply:
I see ambiguity with saying they can assign a number to stats, but not defining the quantity of stats that they assign to. For example, -2 to 5 chosen stats, -2 to infinite chosen stats, or -1 to 2 chosen stats with the ability to use the -1 twice on the same stat, etc.